src/components/Forms/CommentInput/CommentInput.js

```diff
@@ -28,6 +28,8 @@ const preparePayload = (method, data) => {
 };
 
 export const CommentInput = (props) => {
+  const INPUT_CHAR_LIMIT = 1000;
```

Comment:
Should be 2000, our database model restricts the input size to be 2000.

Reply:
The model for [comment (line 23) ](https://github.com/jinchen1036/rdb-crud-microservice/blob/develop/src/models/comments.js) states the limit is 1000.

Reply:
Got it, it's closed